src/scenes/gameplay-scene.ts

```diff
@@ -541,6 +548,7 @@ export class GameplayScene {
   };
 
   public dispose = () => {
+    this.timeTicker.style.display = "none";
```

Comment:
Can we create a dispose method from the Time Ticker component and called it here? So that anything related for clearing or resetting anything related to the time tracker will be within the class itself. 

This will also promote separation of concern has gameplay doesn't necessary need to handle the css of time ticker but just needs to calls the dispose method of time ticker class.